k8s-daemonset/k8s/hello-world.yml

```diff
@@ -76,30 +76,22 @@ spec:
       - name: service
         image: buoyantio/helloworld:0.0.1
         env:
-        - name: POD_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.name
         - name: POD_IP
           valueFrom:
             fieldRef:
               fieldPath: status.podIP
-        - name: NS
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
         command:
         - "/bin/bash"
         - "-c"
-        - "HTTP_PROXY=`./hostIP.sh`:4140 python world.py"
+        - "python world.py"
         ports:
         - name: service
           containerPort: 7778
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: world
+  name: world-v1
```

Comment:
Heh, I don't think so.  I've updated the outgoing dtab in all of the linkerd*.yml files to include the override: `/host/world => /srv/world-v1;`.  The motivation for this change is that as the service mesh series evolves we're going to show how to continuously deploy the world service, in which case we'll need the dtab / service names to have the notion of service versions.  I felt like this was a small enough tradeoff in complexity to warrant making it everywhere, rather than having different service names in different setups.